apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.story.jsx

```diff
@@ -41,7 +41,13 @@ const createStoreWithoutLessonPlan = () => {
   });
   const store = createStoreWithReducers();
   store.dispatch(
-    init([sampleActivityForLessonWithoutLessonPlan], searchOptions, [], false)
+    init(
+      [sampleActivityForLessonWithoutLessonPlan],
+      searchOptions,
+      [],
+      false,
+      false
```

Comment:
this seems like one too many unnamed boolean arguments. can you try turning these into named parameters?
```suggestion
      {
        lessonExtrasAvailableForUnit: false,
        isProfessionalLearningCourse: false
      }
```